Rakefile.rb

```diff
@@ -61,13 +91,13 @@ def name_from(json)
   end
 
   data, _ = json_from('countries.json') rescue {}
+  commit_metadata = file_to_commit_metadata
```

Comment:
The only issue with that change that I can see is that if you're rebuilding everything, the `git log` is about 4 times slower with each country directory listed individually rather than just the `data` directory, so I've changed it to just pass the `data` directory if `to_build == 'data'`. That change is in 3dd27b21106a08.